Core.ApplicationServices/ExcelService.cs

```diff
@@ -496,6 +506,31 @@ public void ImportOrganizationUnits(Stream stream, int organizationId)
             }
         }
 
+        public Result<int, OperationError> ResolveOrganizationIdAndValidateAccess(Guid organizationUuid)
+        {
+            return ResolveOrganizationId(organizationUuid)
+                .Bind(organizationId => AllowAccess(organizationId)
+                    ? Result<int, OperationError>.Success(organizationId)
+                    : new OperationError("User is not allowed to perform batch import", OperationFailure.Forbidden));
+        }
+
+        private Result<int, OperationError> ResolveOrganizationId(Guid organizationUuid)
+        {
+            var organizationId = _entityIdentityResolver.ResolveDbId<Organization>(organizationUuid);
```

Comment:
I think this can be shortened to a match call on _entityIdentityResolver.ResolveDbId